app.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+require 'sinatra'
+require 'sinatra/reloader'
+require 'json'
+require 'erb'
+
+DATA_STORE_PATH = 'data/memo.json'
```

Comment:
```suggestion
DATA_PATH = 'data/memo.json'
```

好みの問題ですが、これぐらいの名前でも十分かも

Reply:
STOREは冗長だったように思えました。
[データをidと値のセットで保持する方式に変更](https://github.com/s-tone-gs/sinatra-memoapp/pull/1/commits/8e80436ca292c228686997f09147ce10e87bf32f)にて対応しました！